hack/examples/overlays/clusters/aws/cilium/helm-addon/kustomization.yaml.tmpl

```diff
@@ -14,3 +14,6 @@ patches:
   - target:
       kind: Cluster
     path: ../../../../../patches/cilium.yaml
+  - target:
+      kind: Cluster
+    path: ../../../../../patches/skip-kube-proxy.yaml
```

Comment:
@dkoshkin Only adding this to the HelmAddon strategy cluster examples.

CRS generation set the K8S_SERVICE_HOST literally to `auto`. When using
the Helm chart, `auto` looks up the host from the `cluster-info` ConfigMap
but this is not available when generating the CRS manifests. To use this,
we would need to update the CRS templates when applying them to the cluster
but we can defer that for now.